docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -581,7 +581,14 @@ Returns file level QC information. It will return a JSON object of the form:
         "File" : $Filename
     },
     "QC" : "Pass|Fail",
-    "Selected" : boolean
+    "Selected" : boolean,
+    "Caveats" : {
+      "Severity" : $severity,
```

Comment:
Wouldn't severity always be "Warn"? If it was excluded it would have never made it into the files table in order to be accessible for the API.

Reply:
Right... I can write 'warn' in the doc instead of $severity, although, it probably does not hurt to be generic

Reply:
I meant you could just remove "Severity" from the response entirely, it doesn't tell the user anything, but I guess it doesn't hurt to keep it either.